src/models/site/model_site.py

```diff
@@ -0,0 +1,16 @@
+from src import db
+from src.models.model_base import ModelBase
+
+
+class SiteModel(ModelBase):
+    __tablename__ = 'sites'
+    uuid = db.Column(db.String(80), primary_key=True, nullable=False)
+    name = db.Column(db.String(80), unique=True, nullable=False)
+    address = db.Column(db.String(80), nullable=True)
+    city = db.Column(db.String(80), nullable=True)
+    state = db.Column(db.String(80), nullable=True)
+    zip = db.Column(db.String(80), nullable=True)
+    country = db.Column(db.String(80), nullable=True)
+    lat = db.Column(db.String(80), nullable=True)
+    lon = db.Column(db.String(80), nullable=True)
+    time_zone = db.Column(db.String(80), nullable=True)
```

Comment:
On a different note, this all flow means: multiple users are associated with an individual site as well. So, if you delete a site associated with a certain user. You can't able to delete `site` until or unless you delete the relation at first. So include this one as well:

`users = db.relationship('UserSiteModel', cascade="all,delete", backref="site", lazy=True)`